src/com/tigersale/userInterface/inventoryManager/RemoveProductView.java

```diff
@@ -0,0 +1,29 @@
+package com.tigersale.userInterface.inventoryManager;
+
+import com.tigersale.database.ProductTable;
+import com.tigersale.model.InventoryManager;
+import com.tigersale.userInterface.AbstractView;
+
+import java.util.Scanner;
+
+/**
+ * @author James Haller
+ */
+public class RemoveProductView extends AbstractView {
+
+    public RemoveProductView(Scanner scanner) {
```

Comment:
There is an EditProductsView which edits description, stock, and price. Should I allow an InventoryManager to edit all product info?